dev.js

```diff
@@ -1,22 +1,33 @@
-const nodemon = require("nodemon");
-const ngrok = require("ngrok");
-const opn = require("opn");
+const nodemon = require('nodemon');
+const ngrok = require('ngrok');
+const opn = require('opn');
 
-require("dotenv").config();
+require('dotenv').config();
 
 const port = process.env.PORT;
+if (
+  process.env.ENVIRONMENT === 'production' ||
+  process.env.NODE_ENV === 'production'
```

Comment:
Do we need to have logic for a product mode? I'd hope that no one tries to turn this workshop into an actual product.

If we do then we should go all the way and use something like [pm2](https://www.npmjs.com/package/pm2) instead of nodemon to keep the process alive.

Reply:
True. This was mainly to not have ngrok start if you import the project into glitch: https://medium.com/glitch/import-code-from-anywhere-83fb60ea4875

An alternative would be to simply change the npm start script to `node backend/server.js`. Maybe that's better. That way it runs seamlessly on glitch.com without starting nrgok. Wdyt?

Reply:
I think it would be easier to just tell people to change the start script after importing into Glitch. I worry that seeing production environment variables might confuse people.

Reply:
@paulasjes-stripe updated, PTAL